01_fib/leonarj4/main_test.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import "testing"
+
+func TestFib(t *testing.T){
+
+	for i:= 0; i <= 7; i++ {
+		
+		switch total := fib(i)
+		total {
+		case 1,3,5,7,9,11,13:
+			t.Logf("The test passed with current number: %d", total)
+		default:
+			t.Errorf("The test did not pass with current number: %d", total)
+		}
```

Comment:
You are only testing your fib() function here but you should also test your main function. 
I also agree with comment above - that you should implement fib() and main() as described in the lab spec.